### PR TITLE
Fix spelling in README.md

### PR DESCRIPTION
**Pull request by MalbaCato**
And other misc. style changes to make it easier to read.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+You can regenerate health like usual with a full hunger bar, so this is **not** UHC for you.
 
 ## Rules
 1. There are no rules  
-Do anything you can think of to gain an advantage. This may include trapping the obsidian platform in the end with lava. Just have a water bucket selected in your toolbar and be ready to run, I guess :wink:. The game will fill a certain area above the obsidian platform with air whenerver a player goes through, so you will have a sort of grace periode.  
+Do anything you can think of to gain an advantage. This may include trapping the obsidian platform in the end with lava. Just have a water bucket selected in your toolbar and be ready to run, I guess :wink:. The game will fill a certain area above the obsidian platform with air whenerver a player goes through, so you will have a sort of grace period.  
 Of course, you as a group can agree on any rules you seem fit.
 
-It is recommended to have more experienced players join the runners and less so the hunters.
+It is recommended to have the more experienced players join the Runners and The less so The Hunters.
```
Comment (NeunEinser) [inline README.md]: > The less so

In that case `the` should be lowercase. Maybe also add "experienced players" after less so? The sentence sounds kinda weird like that imo.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+In case no Runner is in your current dimension, it will still point to the last known location of the runner that was closest to you. Runner locations are saved for you per dimension, so even if you switch a dimension, the compass will still point you to the runner you tracked last in that dimension.
+All of this works with multiple hunters. The compass will track the closest runner for each Hunter.
 
 ## Runners
-As a runner, your goal is to kill the Ender Dragon.  
-It doesn't matter who kills it, so even if it receives its killing blow from a hunter, it counts as a win towards the Runners.  
-You only have one live. If you die, may it be a pvp kill by a hunter, by pve, or through enviornmental damage, you are out. If you play this with multiple runners, you as a team can still win.  
-You can still regenerate health like usual with a full hunger bar, so this is **not** UHC for you.
+As a Runner your goal is to kill the Ender Dragon.  
+It doesn't matter who kills it, even if it receives it's killing blow from a Hunter, it counts as a win towards The Runners.
```
Comment (NeunEinser) [inline README.md]: "Its" was correct. It's not "it is".

**Review by NeunEinser — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (MalbaCato) [reply README.md]: I hate english. You're right.

File: README.md (hunk 1 above)
Comment (NeunEinser) [reply README.md]: Or with experienced players it should be "less experienced" without "so" :D

Comment (MalbaCato) [reply README.md]: Whoops changed the wrong "the". Will fix as soon as I figure out gh's UI again.
I saw the sentence was written badly, but decided against changing more than a few words. Now should be fine.

**Review by NeunEinser — APPROVED** [review]

**Pull request merged**